test/public/defaults.js

```diff
@@ -271,6 +271,32 @@ module.exports.checkColumnBalloon = async (page, rowIndex, columnIndex) => {
     expect(await getInnerHtml(balloon)).to.be.equal(await getInnerHtml(actualContent));
 };
 
+/**
+ * Check that the fourth cell of the given column displays the correct color depending on the status
+ *
+ * @param {{$: function}} page the puppeteer page
+ * @param {number} rowIndex the index of the row to look for status color
+ * @param {number} columnIndex the index of the column to look for status color
+ * @returns {Promise<Chai.Assertion>} void promise
+ */
+module.exports.checkStatusColor = async (page, rowIndex, columnIndex) => {
```

Comment:
I think you should specify that this refers to environment status, in comment and in the function name